misc-utils/kill.1

```diff
@@ -120,7 +121,24 @@ then it can obtain this data via the
 field of the
 .I siginfo_t
 structure.
-
+.TP
+\fB\-\-timeout\fR \fImilliseconds signal\fR
+Send a signal defined the usual way to a process.
+.B \-\-timeout
```

Comment:
We need a better justification here. At first glance, it seems that the option reimplements "kill; sleep; kill" sequence. It would be nice to explain that --timeout guarantee that follow-up signal is without race. Something like:

 "Send a follow-up signal after a timeout without the possibility of race."

... or so ;-)